kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/SchemaRegistryService.java

```diff
@@ -265,11 +252,15 @@ private SchemaSubjectDTO withSchemaType(SchemaSubjectDTO s) {
 
   public Mono<CompatibilityLevelDTO> getSchemaCompatibilityLevel(KafkaCluster cluster,
                                                                  String schemaName) {
-    String configEndpoint = Objects.isNull(schemaName) ? "/config" : "/config/{schemaName}";
+    String globalConfig = Objects.isNull(schemaName) ? "/config" : "/config/{schemaName}";
+    final var values = new LinkedMultiValueMap<String, String>();
+    values.add("defaultToGlobal", "true");
```

Comment:
[Get](https://docs.confluent.io/platform/current/schema-registry/develop/using.html#show-compatibility-requirements-in-effect-for-a-subject) global compatibility level if nothing is specified for the subject